frontends/web/src/components/groupedaccountselector/groupedaccountselector.tsx

```diff
@@ -65,7 +66,7 @@ const SelectSingleValue: FunctionComponent<SingleValueProps<TOption>> = (props)
   );
 };
 
-const SelectOption: FunctionComponent<OptionProps<TOption>> = (props) => {
+const SelectOption = (props: OptionProps<TOption, false>) => {
```

Comment:
why is this needed?

Reply:
I only wanted to remove the `FunctionComponent` as it's not needed (easier to read without it, right?). I'll omit the `false` as thats the default value anyway (for `isMulti` - for multi select in react-select).